From b2323cb36d2b6c6dc5fe064c5dedfb1264179283 Mon Sep 17 00:00:00 2001
From: Carlo Landmeter <clandmeter@gmail.com>
Date: Wed, 25 Jun 2014 11:19:15 +0200
Subject: [PATCH] scripts: BusyBox compatibility

BusyBox killall does not support long options using short options instead
---
 scripts/zmdc.pl.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/zmdc.pl.in b/scripts/zmdc.pl.in
index c891553..579e093 100644
--- a/scripts/zmdc.pl.in
+++ b/scripts/zmdc.pl.in
@@ -784,14 +784,14 @@ sub killAll
     sleep( $delay );
     foreach my $daemon ( @daemons )
     {
-        my $cmd = "killall --quiet --signal TERM $daemon";
+        my $cmd = "killall -q -s TERM $daemon";
         Debug( $cmd );
         qx( $cmd );
     }
     sleep( $delay );
     foreach my $daemon ( @daemons )
     {
-        my $cmd = "killall --quiet --signal KILL $daemon";
+        my $cmd = "killall -q -s KILL $daemon";
         Debug( $cmd );
         qx( $cmd );
     }
-- 
2.0.0

